docs/mobile-apps/automated-testing/ipa-files.md

```diff
@@ -8,15 +8,13 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-We use a [resigning process](#sauce-labs-resigning-enablements) to embed our own provisioning profile into the uploaded `.ipa` file. With the profile in place, the app will be allowed to install and launch on cloud devices. The functionality of the app itself remains completely unmodified.
-
-You can also use `.ipa` files generated by tools outside of Xcode.
+:::note
 
-## Creating .ipa Files for Appium Testing
+To install your iOS app on our cloud devices we need to include our own provisioning profile in your app. We use a [resigning process](#sauce-labs-resigning-enablements) to include our provisioning profile in your app. While this process does not modify the functionality of the app, it touches [certain entitlements in your app](#sauce-labs-resigning-enablements). It is possible to disable the resigining process by buying your own private devices with dedicated provisioning profiles, see [here](#private-devices).
```

Comment:
lets add here real device instead of cloud device.

Reply:
We need to call out here two more things:
1. You can disable only if the app is ad-hoc and you embed the private device UDID. 
2. You can disable resigning/instrumentation with an enterprise certificate without the need of adding the UDID to your provisioning profle